### DISCUSSION TITLE
How to delete old universal abbreviations

### DISCUSSION THREAD

**Discussion by nap2001** [fish-shell/fish-shell#9467] [Q&A] [answered]
Since release 3.6.0's enhancements to abbreviations, I need to rethink/redesign/cleanup my abbreviations. How do I get rid of my pre-3.6.0 universal abbreviations? I've tried "abbr -e" but that only deletes in the current shell and new shells still have the deleted old abbreviation. I have 105 universal abbreviations and think I should just start over given changes in 3.6.0.

**Comment by faho** [comment, marked as answer]
```fish
echo set -e (set -Un | string match -re '^_fish_abbr_')
```

remove the `echo` if you're happy.

I opened #9468 to track making `abbr -e` work.
